src/Chainweb/Version.hs

```diff
@@ -782,7 +794,8 @@ twentyChainUpgrade
     -> ChainId
     -> BlockHeight
     -> Bool
-twentyChainUpgrade Mainnet01 _ h = h == maxBound -- TODO: update me with a calculated height
+twentyChainUpgrade Mainnet01 _ h = h == to20ChainsMainnet
+twentyChainUpgrade Testnet04 _ h = h == to20ChainsTestnet
```

Comment:
```suggestion
twentyChainUpgrade Mainnet01 _ h = h == 771_474 -- ~ 2020-07-23 16:30:00
twentyChainUpgrade Testnet04 _ h = h == 318_264 -- ~ 2020-07-23 16:30:00
```

Reply:
See previous comment about importance of pairing with transition